cranelift-codegen/meta/src/cdsl/formats.rs

```diff
@@ -49,15 +49,15 @@ pub(crate) struct InstructionFormat {
 pub(crate) struct FormatStructure {
     pub num_value_operands: usize,
     pub has_value_list: bool,
-    pub imm_field_names: Vec<&'static str>,
+    pub imm_field_names: Vec<(&'static str, &'static str)>,
```

Comment:
Might be good to make clear, through a doc string or by type-aliasing, that this tuple is `(Rust field name, Rust type)`